connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/PluginUtils.java

```diff
@@ -128,6 +128,12 @@
             + "|file\\..*"
             + "|converters\\..*"
             + "|storage\\.StringConverter"
+            + "|storage\\.SimpleHeaderConverter"
```

Comment:
nit: Let's add it in the end of this list, in the order that appears in the unit test which also happens to be the lexicographic order (also a header converter and not a regular converter).